requirements-dev.txt

```diff
@@ -6,16 +6,15 @@ pytz>=2015.6
 jsonschema>=2.5.1
 altair>=3.1.0
 ruamel.yaml>=0.15.24
-ipywidgets>=7.4.2
+ipywidgets>=7.5.1
```

Comment:
One other pre-commit hook you might want to install is `requirements-txt-fixer` -- automatically sorts your requirements, it's minor but quite nice 😀 -- see below for an example

f7a20c31fcc820b7ed3a894ae9e6/.pre-commit-config.yaml#L20

Reply:
Added!